README.md

```diff
@@ -116,6 +116,9 @@ Global parameters apply to all sub-charts and make it easier to configure resour
 | `global.gitlabEnterprise.hostname`              | Hostname of gitlabEnterprise instance                                                                        | `nil`           |
 | `global.gitlabEnterprise.protocol`              | Protocol of gitlabEnterprise instance                                                                        | `nil`           |
 | `global.gitlabEnterprise.port`                  | Port of gitlabEnterprise instance                                                                            | `nil`           |
+| `global.bitbucket.enabled`                      | Enable bitbucket                                                                                             | `nil`           |
```

Comment:
I know this is also not consistent for the other Git providers, but here we should capitalize the name of the Git provider correctly:

```suggestion
| `global.bitbucket.enabled`                      | Enable Bitbucket                                                                                             | `nil`           |
```